src/main.rs

```diff
@@ -26,6 +28,7 @@ fn main() {
     opts.optopt("R", "", "changes the git remote used (default is origin)", "REMOTE");
     opts.optopt("b", "", "changes the base for merged branches (default is master)", "BRANCH");
     opts.optflag("h", "help", "print this help menu");
+    opts.optflag("", "version", "print the version");
```

Comment:
I hadn't added it due to the potential confusion that `-v` sometimes is verbose but I can add it if you want!